src/box/iproto_features.c

```diff
@@ -14,6 +14,9 @@
 
 const char *iproto_feature_id_strs[iproto_feature_id_MAX] = {
 	IPROTO_FEATURES(IPROTO_FEATURE_ID_STRS_MEMBER)
+#if defined(ENABLE_FETCH_SNAPSHOT_CHECKPOINT_CURSOR)
+	IPROTO_FEATURE_ID_STRS_MEMBER(FETCH_SNAPSHOT_CHECKPOINT_CURSOR, 10)
+#endif /* defined(ENABLE_FETCH_SNAPSHOT_CHECKPOINT_CURSOR) */
```

Comment:
IMO the name is awkward because "checkpoint" and "snapshot" are synonyms in Tarantool. I still think `ENABLE_FETCH_SNAPSHOT_CURSOR` is a better name but I'm okay with it if other reviewers think it's okay.

Reply:
I don't care that much about feature's naming. Let's use `ENABLE_FETCH_SNAPSHOT_CURSOR`